src/public/index.html

```diff
@@ -15,6 +15,12 @@
             align-items: center;
         }
     </style>
+    <meta property="og:title" content="heartpoints.org">
+    <meta property="og:type" content="website">
+    <meta property="og:description" content="the currency of good">
+    <meta property="og:image" content="/images/heartpointsIcon.png">
+    <meta property="og:url" content="www.heartpoints.org">
```

Comment:
wondering if this should have http:// in it or not? any idea? maybe it works like this and is better to not specify? not sure.